developer/src/kmldmlc/test/test-loca.ts

```diff
@@ -18,13 +18,16 @@ describe('loca', function () {
   it('should compile multiple locales', function() {
     const callbacks = new CompilerCallbacks();
     let loca = loadSectionFixture(LocaCompiler, 'sections/loca/multiple.xml', callbacks) as Loca;
-    assert.equal(callbacks.messages.length, 0);
 
     // Note: multiple.xml includes fr-FR twice, with differing case, which should be canonicalized
+    assert.equal(callbacks.messages.length, 1);
+    assert.deepEqual(callbacks.messages[0], CompilerErrors.OneOrMoreRepeatedLocales());
+
+    // Original is 6 locales, now five minimized in the results
     assert.equal(loca.locales.length, 5);
     assert.equal(loca.locales[0], 'mt');
-    assert.equal(loca.locales[1], 'fr-FR');
-    assert.equal(loca.locales[2], 'km-Khmr-KH');
+    assert.equal(loca.locales[1], 'fr');  // Original fr-FR
+    assert.equal(loca.locales[2], 'km');  // Original km-Khmr-kh
```

Comment:
```suggestion
    assert.equal(loca.locales[2], 'km');  // Original km-Khmr-KH
```

Reply:
See https://github.com/keymanapp/keyman/blob/7d38e26b09dcf06e00f316cd54cfa9a2fe4b41e1/developer/src/kmldmlc/test/fixtures/sections/loca/multiple.xml#L7 :) it's lower case in the source file!

Reply:
oh ok.  i was going by the above line.